src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Emitter.cs

```diff
@@ -719,19 +698,39 @@ private static void EmitSimplifiedGo(IndentedTextWriter writer, RegexMethod rm,
             int nextLocalId = 0;
             string GetNextLocalId() => $"i{nextLocalId++}"; ;
 
-            RegexNode node = rm.Tree.Root;
+            RegexNode node = rm.Code.Tree.Root;
             Debug.Assert(node.Type == RegexNode.Capture, "Every generated tree should begin with a capture node");
             Debug.Assert(node.ChildCount() == 1, "Capture nodes should have one child");
 
             // Skip the Capture node. We handle the implicit root capture specially.
             node = node.Child(0);
 
+            // In some limited cases, FindFirstChar will only return true if it successfully matched the whole thing.
+            // This is the case, in particular, for strings.  We can special case these to do essentially nothing
```

Comment:
what does 

> This is the case, in particular, for strings.

mean? Did you mean to describe the kind of strings here?

Reply:
I could have used the word "multis" if that would be clearer, e.g. in `[ab][cd]efghi[jk]`, the "efghi" sequence of multiple characters next to each other.